CHANGELOG.rst

```diff
@@ -18,6 +18,7 @@ Changed
 - Hid ui primary and secondary constraints on ``k-toolbar`` in the meantime
 - Moved request circuit ``k-button`` out of k-accordion-item since it's mandatory
 - The traces being check rely on ``type``: ``last`` to be considered valid.
+- ``dl_vlan`` value is mapped to an integer in range [1, 4095] for the ``/traces``requests to ``sdntrace_cp``. 
```

Comment:
Looks like the extra backticks formatting as code ended up including more words than it should